config/initializers/devise.rb

```diff
@@ -105,7 +105,7 @@
   # able to access the website for two days without confirming his account,
   # access will be blocked just in the third day. Default is 0.days, meaning
   # the user cannot access the website without confirming his account.
-  # config.allow_unconfirmed_access_for = 2.days
+  config.allow_unconfirmed_access_for = 2.days
```

Comment:
So we don't want to get of the confirmation as Jos suggested?
Anyway, I think this is a good compromise...